src/command/backup/backup.c

```diff
@@ -989,7 +989,7 @@ backupFilePut(BackupData *backupData, Manifest *manifest, const String *name, ti
 
             manifestFileAdd(manifest, &file);
 
-            LOG_DETAIL_FMT("wrote '%s' file returned from pg_stop_backup()", strZ(name));
+            LOG_DETAIL_FMT("wrote '%s' file returned from backup", strZ(name));
```

Comment:
As this is no longer clear this is a function by context, maybe say `returned from backup function`

Same for other lines below e.g. 2960

Reply:
Good point, fixed in a9e546c3dc90.